terraform/helm/aptos-node/files/configs/fullnode-base.yaml

```diff
@@ -11,8 +11,6 @@ execution:
   genesis_file_location: /opt/aptos/genesis/genesis.blob
 
 storage:
-  rocksdb_configs:
```

Comment:
we are removing this? will this disable sharding?

Reply:
Seems like that would undo this 

https://github.com/aptos-labs/aptos-core/commit/8f5b5b209e1f51abd08f3bf3f3116be95ca06ca3